### PR TITLE
Remove dead MIT gpg site and fix release command

### PR DESCRIPTION
**Pull request by Superskyyy**
1. MIT gpg site is gone, replaced with ubuntu keyserver.
2. Release command was previously wrong, now fixed.

- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue url. Closes: <URL to main repo issue>
- [ ] Update the [`CHANGELOG.md`](https://github.com/apache/skywalking-python/blob/master/CHANGELOG.md).

### CODE REVIEW

File: docs/en/contribution/How-to-release.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ This documentation guides the release manager to release the SkyWalking Python i
 
 ## Add your GPG public key to Apache svn
 
-1. Upload your GPG public key to a public GPG site, such as [MIT's site](http://pgp.mit.edu:11371/). 
+1. Upload your GPG public key to a public GPG site, such as [Ubuntu's GPG keyserver](https://keyserver.ubuntu.com/).
```
Comment (wu-sheng) [inline docs/en/contribution/How-to-release.md:13]: Is this an official rename?

Comment (wu-sheng) [reply docs/en/contribution/How-to-release.md:13]: Or simply mit one is gone? 
The reason I asked is about Maven release, the nexus will use some gpg servers to access public keys for verification. This is why it is mentioned in Java release, and copied here.

Comment (Superskyyy) [reply docs/en/contribution/How-to-release.md:13]: > Or simply mit one is gone? The reason I asked is about Maven release, the nexus will use some gpg servers to access public keys for verification. This is why it is mentioned in Java release, and copied here.

I think the MIT one is gone and won't come back any more. Last time I did Python agent release, the MIT site was already broken so I used the Ubuntu keyserver and it worked perfectly.

Comment (kezhenxu94) [reply docs/en/contribution/How-to-release.md:13]: Uploading gpg key to a public server is not required in Python agent release process, let's simply remove this in the doc @Superskyyy

Comment (Superskyyy) [reply docs/en/contribution/How-to-release.md:13]: Sure!

File: Makefile [hunk 2]
```diff
@@ -130,6 +130,6 @@ clean:
 
 .PHONY: release
 release: clean lint license
-	-tar -zcvf skywalking-python-src-$(VERSION).tgz --exclude .venv *
-	gpg --batch --yes --armor --detach-sig skywalking-python-src-$(VERSION).tgz
-	shasum -a 512 skywalking-python-src-$(VERSION).tgz > skywalking-python-src-$(VERSION).tgz.sha512
+	tar -zcvf skywalking-python-src-$VERSION.tgz --exclude .venv *
+	gpg --batch --yes --armor --detach-sig skywalking-python-src-$VERSION.tgz
```
Comment (Superskyyy) [inline Makefile:134]: So you mean this line should also be removed right?@kezhenxu94

File: docs/en/contribution/How-to-release.md [hunk 3]
```diff
@@ -7,16 +7,6 @@ This documentation guides the release manager to release the SkyWalking Python i
...
 2. Update CHANGELOG.md and `version` in `pyproject.toml`.
 
-
-## Add your GPG public key to Apache svn
-
-1. Upload your GPG public key to a public GPG site, such as [MIT's site](http://pgp.mit.edu:11371/). 
-
-1. Log in [id.apache.org](https://id.apache.org/) and submit your key fingerprint.
-
-1. Add your GPG public key into [SkyWalking GPG KEYS](https://dist.apache.org/repos/dist/release/skywalking/KEYS) file, **you can do this only if you are a PMC member**.  You can ask a PMC member for help. **DO NOT override the existed `KEYS` file content, only append your key at the end of the file.**
```
Comment (kezhenxu94) [inline docs/en/contribution/How-to-release.md:17]: This step is needed however, the vote and the users need this to check the tars are signed by the release manager.

File: docs/en/contribution/How-to-release.md (hunk 1 above)
Comment (kezhenxu94) [reply docs/en/contribution/How-to-release.md]: > Uploading gpg key to a public server

The other step "upload the key to Apache svn " is still needed 🤭

File: docs/en/contribution/How-to-release.md (hunk 3 above)
Comment (Superskyyy) [reply docs/en/contribution/How-to-release.md:17]: Fixed!

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**